crates/bevy_app/src/plugin_group.rs

```diff
@@ -2,7 +2,9 @@ use crate::{App, Plugin};
 use bevy_utils::{tracing::debug, HashMap};
 use std::any::TypeId;
 
+/// Combines multiple [`Plugin`]s into a single unit.
 pub trait PluginGroup {
+    /// Configures the [Plugin]s that are to be added.
```

Comment:
```suggestion
    /// Configures the [`Plugin`]s that are to be added.
```

Style nit: these will show up with nicer code-style formatting in the generated docs.